.github/workflows/reusable-release.yml

```
@@ -91,64 +110,183 @@ jobs:
           version_suffix=$version_suffix
           EndOfFile
 
+
   generate-version-update-commits:
-    name: 🎊 Test & Version
-    needs: init
+    name: 🤸🏽 Generate version update commits
```

Deliberately not including `test` here. Gradle/Android libraries have a variety of tests - jvm unit, instrumented android, screenshots, and end to end. This being a reusable workflow, the test responsibility is left to individual repos to set up, as needed, as a previous job before invoking this workflow.

---

Excluding the tests makes sense, see:

* https://github.com/guardian/gha-scala-library-release-workflow/issues/42

Could we keep emoji language consistent tho? `🎊` not `🤸🏽`! 🤣 

I suggest:

```suggestion
    name: 🎊 Versioning
```

Long job names get truncated in the GitHub Actions UI, so it's nice to go shorter. 'Versioning' is also a name that we could probably adopt on `gha-scala-library-release-workflow` too.